nimbus-ui/nimbusui/src/app/components/platform/validators/custom.validators.ts

```diff
@@ -74,6 +75,16 @@ export class CustomValidators {
         };
     }
 
+    static isNotEmpty(control: FormControl){
+        if (control.value){
```

Comment:
Shouldn't we also be checking to ensure `control.value` does not contain only spaces?